pkg/cmd/kubernetes_exec.go

```diff
@@ -219,45 +227,17 @@ func helmInit() error {
 	return nil
 }
 
-func kubectlTask(parts ...string) (execute.ExecResult, error) {
-	task := execute.ExecTask{
-		Command: "kubectl",
-		Args:    parts,
+func kubectl(kubeConfigPath string, kubeContext string, parts ...string) execute.ExecTask {
+	if len(kubeConfigPath) > 0 {
+		parts = append(parts, "--kubeconfig")
+		parts = append(parts, kubeConfigPath)
 	}
-
-	res, err := task.Execute()
-
-	return res, err
-}
-
-func kubectl(parts ...string) error {
```

Comment:
Are you dropping this method completely now?

Reply:
yes there is only one method for now which return ExecTask